common/protos-java-kt/README.md

```diff
@@ -0,0 +1,22 @@
+# Compiled Protos for Java and Kotlin
+
+## Steps to publish:
+
+1) Create a Personal Access Token from Github with write/read/delete access to packages.
+2) Create a copy of `artifactory.properties.template` as `artifactory.properties`.
+3) Replace <GITHUB Email> with your email id for github.
+3) Replace <GITHUB Personal Access Token> with your personal access token.
+4) Run `make publish` to publish package to github packages.
```

Comment:
Add a note that this will be published to one's respective fork. To publish as a hyperledger-labs package, a PR will need to be committed and merged.